<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cronjobs</class>
 <widget class="QWidget" name="cronjobs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>436</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Stella Actions</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="stellaActions"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="2">
         <widget class="QSpinBox" name="spinChannel">
          <property name="maximum">
           <number>7</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QSpinBox" name="spinValue">
          <property name="maximum">
           <number>255</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="label_2">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Value:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>spinValue</cstring>
          </property>
         </widget>
        </item>
        <item row="1" column="3" colspan="2">
         <widget class="QSlider" name="horizontalSlider">
          <property name="maximum">
           <number>255</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="btnAddAction">
          <property name="toolTip">
           <string>Refresh cronjobs view</string>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
          <property name="icon">
           <iconset resource="../../app.qrc">
            <normaloff>:/icons/document-new.png</normaloff>:/icons/document-new.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="btnRemoveAction">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string>Remove selected cronjob</string>
          </property>
          <property name="text">
           <string>Remove</string>
          </property>
          <property name="icon">
           <iconset resource="../../app.qrc">
            <normaloff>:/icons/edit-delete.png</normaloff>:/icons/edit-delete.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QComboBox" name="spinFunction"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>New cron job</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Hour:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>spinHour</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinHour">
        <property name="minimum">
         <number>-23</number>
        </property>
        <property name="maximum">
         <number>23</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Minute:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>spinMinute</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinMinute">
        <property name="minimum">
         <number>-59</number>
        </property>
        <property name="maximum">
         <number>59</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Day:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>spinDay</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinDay">
        <property name="minimum">
         <number>-31</number>
        </property>
        <property name="maximum">
         <number>31</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Month:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>spinMonth</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinMonth">
        <property name="minimum">
         <number>-12</number>
        </property>
        <property name="maximum">
         <number>12</number>
        </property>
        <property name="value">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_9">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>Weekday:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>spinDayofweek</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QComboBox" name="spinDayofweek"/>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Repeat:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="buddy">
         <cstring>spinRepeat</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="spinRepeat">
        <property name="minimumSize">
         <size>
          <width>90</width>
          <height>0</height>
         </size>
        </property>
        <property name="specialValueText">
         <string>Unlimited</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QPushButton" name="btnAddCronjob">
        <property name="text">
         <string>Add cronjob</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QPushButton" name="btnCalendar">
        <property name="toolTip">
         <string>Choose date with calendar</string>
        </property>
        <property name="text">
         <string>Calendar</string>
        </property>
        <property name="icon">
         <iconset resource="../../app.qrc">
          <normaloff>:/icons/view-calendar-day.png</normaloff>:/icons/view-calendar-day.png</iconset>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QPushButton" name="btnNow">
        <property name="toolTip">
         <string>Current time</string>
        </property>
        <property name="text">
         <string>Now</string>
        </property>
        <property name="icon">
         <iconset resource="../../app.qrc">
          <normaloff>:/icons/chronometer.png</normaloff>:/icons/chronometer.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>spinHour</tabstop>
  <tabstop>spinMinute</tabstop>
  <tabstop>spinDay</tabstop>
  <tabstop>spinMonth</tabstop>
  <tabstop>spinDayofweek</tabstop>
  <tabstop>spinRepeat</tabstop>
  <tabstop>spinChannel</tabstop>
  <tabstop>spinValue</tabstop>
  <tabstop>btnAddAction</tabstop>
  <tabstop>btnAddCronjob</tabstop>
  <tabstop>stellaActions</tabstop>
  <tabstop>btnRemoveAction</tabstop>
  <tabstop>horizontalSlider</tabstop>
 </tabstops>
 <resources>
  <include location="../../app.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinValue</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinValue</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
